src/enhance/lintCode.ts

```diff
@@ -0,0 +1,39 @@
+import { GenerationTargetType } from '../getViewForSwagger2';
+
+const lint: any = require('jshint').JSHINT;
+
+export interface LintOptions {
+    esnext: boolean;
+    lint: boolean;
+}
+
+export interface LintError {
+    reason: string;
+    evidence: string;
+    code: string;
+}
+
+export function lintCode(opts: LintOptions, type: GenerationTargetType, source: string): void {
+    const lintOptions = {
+        // TODO: This seems like a weird leftover from the typescript cleanup? We probably don't need to support lint at all?
```

Comment:
+1

Reply:
+1 for removing linting within the generator aswell

Reply:
Added to task #34